builder/proxmox/common/step_start_vm.go

```diff
@@ -69,6 +88,49 @@ func (s *stepStartVM) Run(ctx context.Context, state multistep.StateBag) multist
 		Onboot:       &c.Onboot,
 	}
 
+	if c.PackerForce {
+		ui.Say("Force set, checking for existing VM or template")
+		vmrs, err := getExistingVms(c, client)
+		if err != nil {
+			// This can happen if no VMs found, so don't consider it to be an error
```

Comment:
yes, although it's a pretty sub-optimal check. the proxmox client doesn't have typed errors, so we would have to check if the error string matches either `vm '%d' not found` or `vm '%s' not found` with the VM ID or name filled in.

I have no problem adding this with some logic to just note when nothing is found, but error and fail if there's a different error, although this could break if the client is updated and the error string ever changes